usr/lib/soft_stdll/soft_specific.c

```diff
@@ -224,6 +224,7 @@ CK_RV token_specific_des_ecb(STDLL_TokData_t *tokdata,
                              CK_ULONG *out_data_len,
                              OBJECT *key, CK_BYTE encrypt)
 {
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Comment:
patch title: `Soft: replace dedicated OpenSSL interfaces`
I guess you mean **deprecated**, not **dedicated** ....

Reply:
done